### PR TITLE
Add OpenEXR

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add OpenEXR

### CODE REVIEW

**Comment by Vertexwahn** [comment]
Please add `skip-url-stability-check` tag since there is no release archive URL

File: modules/openexr/3.2.0/patches/module_dot_bazel.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -7,12 +7,3 @@
+    bazel_dep(name = "bazel_skylib", version = "1.4.2")
+    bazel_dep(name = "imath", version = "3.1.9")
+    bazel_dep(name = "libdeflate", version = "1.18")
+-module(
```
Comment (Wyverald) [inline modules/openexr/3.2.0/patches/module_dot_bazel.patch:7]: the patch file is somehow mangled.

**Comment by Vertexwahn** [comment]
@Wyverald Should be fixed now!

**Comment by Vertexwahn** [comment]
@Wyverald Hopefully fixed now ;)

File: modules/openexr/3.2.0/MODULE.bazel [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module(
+    name = "openexr",
+    version = "3.2.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(name = "imath", version = "3.1.9")
```
Comment (Wyverald) [inline modules/openexr/3.2.0/MODULE.bazel:8]: `repo_name = "Imath"`

File: modules/openexr/3.2.0/patches/module_dot_bazel.patch (hunk 1 above)
Comment (Vertexwahn) [reply modules/openexr/3.2.0/patches/module_dot_bazel.patch]: Thanks for the hint - resolved

File: modules/openexr/3.2.0/patches/module_dot_bazel.patch [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+@@ -1,9 +1,10 @@
+ module(
+     name = "openexr",
+-    version = "0.0.0", # The version "0.0.0" is set here and will be patched within the Bazel Central Registry
++    version = "3.2.0",
+     compatibility_level = 1,
+ )
+ 
+ bazel_dep(name = "bazel_skylib", version = "1.4.2")
+ bazel_dep(name = "imath", repo_name = "Imath", version = "3.1.9")
```
Comment (Wyverald) [inline modules/openexr/3.2.0/patches/module_dot_bazel.patch:12]: this line probably needs to be patched (the original file didn't have `repo_name=...`)

Comment (Vertexwahn) [reply modules/openexr/3.2.0/patches/module_dot_bazel.patch]: another iteration...

**Comment by Vertexwahn** [comment]
@Wyverald  Maybe now...?

**Review by Wyverald — APPROVED** [review]
congrats :)

**Pull request merged**